src/presentation/http/router/note.ts

```diff
@@ -85,13 +93,17 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
       return reply.notFound('Note not found');
     }
 
+    const parentId = await noteRelationsService.getParentNoteIdByNoteId(note.id);
+
+    const parentNote = parentId !== null ? await noteService.getNoteById(parentId) : undefined;
     /**
      * Check if current user is creator of the note
      */
     const canEdit = note.creatorId === request.userId;
 
     return reply.send({
       note: note,
+      parentNote: parentNote,
```

Comment:
not covered by tests

Reply:
Such a test case is here https://github.com/codex-team/notes.api/blob/3b5ee272c793062c3f3d3afbb951fc5189c5d078/src/presentation/http/router/note.test.ts#L58